jwt-authorizer/templates/README.md

```diff
@@ -0,0 +1,162 @@
+# Deploying
```

Comment:
I'm a bit confused by these instructions.  Are these the ones you want people to run?

Usually people put in a README.md here and template it out (putting in like the namespace and other full URLs) to help explain what got deployed - not necessarily how to deploy it.

I'm also confused by the init.sh... can't we just have a values.yaml checked in?  That's how we'll need to do it with argocd.

Reply:
This readme was not supposed to be checked in.

The init script generates key pairs, secrets, and a few other things for convenience.

I will rewrite the README.md because it's not very helpful (I didn't mean to check it in). I think the init script needs to change unless there's some way I can figure out how to generate some of the keys/secrets from Helm. It is mostly just spitting out instructions on how to modify the values.yaml file.

Reply:
I've refactored README.md into two files, a part with README.md, which is not in the templates directory, and a NOTES.txt, which is in the templates directory.